ui/components/app/assets/defi-list/defi-list.tsx

```diff
@@ -85,6 +85,7 @@ export default function DefiList({ onClick }: DefiListProps) {
           marketValue: formatCurrencyWithMinThreshold(marketValue, 'USD'),
           chainId: chainId as Hex,
           iconGroup,
+          tokenFiatAmount: marketValue,
```

Comment:
The sorting by value wasn't working, since it relied on this field to be present.